test/extended/networking/external_gateway.go

```diff
@@ -0,0 +1,60 @@
+package networking
+
+import (
+	"context"
+	"net"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/origin/test/extended/util"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	clientset "k8s.io/client-go/kubernetes"
+	frameworkpod "k8s.io/kubernetes/test/e2e/framework/pod"
+)
+
+var _ = g.Describe("[sig-network] external gateway address", func() {
+	oc := exutil.NewCLI("ns-global")
+
+	InOVNKubernetesContext(func() {
+		f := oc.KubeFramework()
+
+		g.It("Should validate failure if an external gateway address does not match the address family of the pod", func() {
```

Comment:
The describe and it together should make a comprehensible phrase

```suggestion
		g.It("should match the address family of the pod", func() {
```